### DISCUSSION TITLE
good extension

### DISCUSSION THREAD

**Discussion by MeeperTheGreat99** [WilliamVenner/vscode-glua-enhanced#63] [General]
So, every addon I've ever made that involves glua was made in a poor fashion I realize. IDE? Nah, this is Garry's Mod, we don't use that fancy stuff, I got Notepad++. Yeah... long story short, I googled something like "gmod lua development environment" and realized all those hours of debugging caused by lack of simple things such as syntax highlighting or even autocomplete could've been solved by just using VSCode and a handful of glua related extensions. Worst part is I already had VSCode and didn't even think of using it for Garry's Mod addon development. Silly me. Thanks for making my day with this

**Comment by WilliamVenner** [comment]
You're welcome!!! :D
